kolibri/core/assets/src/views/immersive-full-screen/index.vue

```diff
@@ -54,7 +54,7 @@
 
   .top-bar
     height: 60px
-    background-color: $core-text-default
+    background-color: $core-action-dark
     box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)
 
   .back-btn
```

Comment:
Make use of the keen large `ui-icon-button` for the back button. The name of the user does not make sense to be `backPageText`

Reply:
That's the text used in the mock up. What should it say instead?

Reply:
So currently, this `immersive-full-screen` component does not match what we have in the style guide. In the style guide there is meant to be a `full-page-modal` which consists of a close icon button and a title. This has no url. Then there are these `end-states` which have urls. In this case, the only thing that changes is that the app-bar(app-wide bar) is replaced/overlayed with an `end-state-bar`, which has with a back button icon link and a title. It is not meant to be a full page component/overlay. 

Full page modal

https://docs.google.com/document/d/15Y1RDzsO_vpRSSEtL_L5lCzZtAtN4j3qN813_bnL4uw/edit#

Action bars

https://docs.google.com/document/d/1oqo8JbiNAWPg0qR1EUsvqG_2QRoMCxKUmGEvnFBRbLo/edit#heading=h.juu4pkeoyu2c

Reply:
Until that bar is  implemented, we'll have to fake it with immersive-full-screen for now.

Probably, the better `backPageText` would be "User Permissions" or something like that.

<- User Permissions